src/main/java/qna/domain/Question.java

```diff
@@ -85,8 +85,37 @@ public boolean isDeleted() {
 		return deleted;
 	}
 
-	public void setDeleted(boolean deleted) {
-		this.deleted = deleted;
+	public List<DeleteHistory> delete(User loginUser) throws CannotDeleteException {
+		if (!this.canDeleteQuestion(loginUser)) {
+			throw new CannotDeleteException("질문을 삭제할 권한이 없습니다.");
+		}
+		setDelete();
+
+		List<DeleteHistory> deleteHistories = addDeleteHistories(loginUser);
+		return deleteHistories;
+	}
```

Comment:
바로 return 해줘도 좋을 것 같아요! 😄 
```suggestion
	public List<DeleteHistory> delete(User loginUser) throws CannotDeleteException {
		if (!this.canDeleteQuestion(loginUser)) {
			throw new CannotDeleteException("질문을 삭제할 권한이 없습니다.");
		}
		setDelete();

		return addDeleteHistories(loginUser);
	}
```

Reply:
길이를 줄일 수 있는 것들을 자꾸 놓치는것같네요ㅠ 수정하였습니다!
감사합니다 😀